README.md

```diff
@@ -4,11 +4,50 @@
 
 ## Quick Start
 
-To use log4cats-mtl in an existing SBT project with Scala 2.11 or a later version, add the following dependencies to your
+To use log4cats-mtl in an existing SBT project with Scala 2.12 or a later version, add the following dependencies to your
 `build.sbt` depending on your needs:
 
 ```scala
 libraryDependencies ++= Seq(
   "io.chrisdavenport" %% "log4cats-mtl" % "<version>"
 )
 ```
+
+## Examples
+
+```tut
```

Comment:
I don't believe the README is passed through the docs, only `docs/XYZ`, as such I think this should be  \```scala

If this was in docs/index.md I think it would be 
\```scala mdoc

Reply:
Ah, my bad. I forgot to add the example to `docs/index.md` as well.  
I've pushed a new version.   
`site / mdoc` works fine right now.